data/state/account_test.go

```diff
@@ -242,3 +243,104 @@ func TestAccount_SetCodeHashWithJournal(t *testing.T) {
 	assert.Equal(t, 1, journalizeCalled)
 	assert.Equal(t, 1, saveAccountCalled)
 }
+
+func TestAccount_SetOwnerAddressWithJournal(t *testing.T) {
+	t.Parallel()
+
+	count := 0
+	tracker := &mock.AccountTrackerStub{
+		JournalizeCalled: func(entry state.JournalEntry) {
+			count++
+		},
+		SaveAccountCalled: func(accountHandler state.AccountHandler) error {
+			count++
+			return nil
+		},
+	}
+
+	acc, _ := state.NewAccount(&mock.AddressMock{}, tracker)
+
+	owner := []byte("owner")
+	err := acc.SetOwnerAddressWithJournal(owner)
+	require.Nil(t, err)
+	require.Equal(t, 2, count)
+	require.Equal(t, acc.OwnerAddress, owner)
+}
+
+func TestAccount_AddAndClaimDeveloperRewards(t *testing.T) {
+	t.Parallel()
+
+	tracker := &mock.AccountTrackerStub{
```

Comment:
instead of providing "empty" implementations for AccountTrackerStub a lot of times, better change that stub this way (for example for Journalize):
``` GOLANG
// Journalize -
func (ats *AccountTrackerStub) Journalize(entry state.JournalEntry) {
    if ats.JurnalizeCalled != nil {
	ats.JournalizeCalled(entry)
    }
}
```

Reply:
done.